integration-tests/src/chains/paseo/genesis.rs

```diff
@@ -1,16 +1,17 @@
 use emulated_integration_tests_common::{
-	accounts, build_genesis_storage, get_account_id_from_seed, get_from_seed, get_host_config,
-	validators,
+	accounts, build_genesis_storage, get_from_seed, get_host_config, validators,
 };
-use paseo_runtime_constants::currency::UNITS as PAS;
 use polkadot_primitives::{AssignmentId, Balance, ValidatorId};
+use polkadot_runtime as runtime;
+use polkadot_runtime_constants as runtime_constants;
+use runtime_constants::currency::UNITS as PAS;
```

Comment:
Can stay the same because we are changing back to PAS

Reply:
Probably not as that dependency no longer exists.